lib/site_prism/waiter.rb

```diff
@@ -12,6 +12,11 @@ def self.wait_until_true(wait_time_seconds = default_wait_time)
       raise SitePrism::TimeoutException, 'Timed out while waiting for block to return true'
     end
 
+    def self.wait_with_capybara(timeout = self.default_wait_time, or_raise: nil)
+      raise or_raise unless (result = Capybara.using_wait_time(timeout) { yield }) || !or_raise
+      result
+    end
+
```

Comment:
Considering renaming `or_raise` to something better...maybe `raise_on_fail`?  Suggestions?